src/BeachesCard.js

```diff
@@ -7,13 +7,32 @@ class BeachesCard extends Component {
   }
 
   render() {
+    const beachProps = this.props.selectedBeach ? Object.keys(this.props.selectedBeach) : []
     if (!this.props.selectedBeach) {
       return ( <div className='none'></div> );
     } else {
     return (
-      <div className='popup'> 
+      <div className='popup'>
+        <button className='pop-btn' onClick={this.props.closeBeach}>X close</button>
+        <img className='popup-img' src={this.props.selectedBeach.image}/>
         <p>{this.props.selectedBeach.name}</p>
         <p>{this.props.selectedBeach.county}</p>
+        <ul>Recommended Activities:
+           {
+            this.props.selectedBeach.activities.map((activity) => {
+              return <li>{activity}</li>
+            })
+           }
+         </ul>
+         <ul>This beach has:
+           {
+            beachProps.map((prop) => {
+               if (this.props.selectedBeach[prop] === true) {
+                 return <li>{prop}</li>
+               }
+            })
+           }
+         </ul>
```

Comment:
Thank you for getting all the data for each beach rendering in the pop-up.  I think we should add the county to the main page with all the beaches.